clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/AbstractAmazonLoadBalancerCachingAgent.groovy

```diff
@@ -129,7 +129,7 @@ abstract class AbstractAmazonLoadBalancerCachingAgent implements CachingAgent, O
   abstract CacheResult loadDataInternal(ProviderCache providerCache)
 
   @Override
-  Collection<Map> pendingOnDemandRequests(ProviderCache providerCache) {
+  Collection<Map<?, ?>> pendingOnDemandRequests(ProviderCache providerCache) {
```

Comment:
I suspect this could be `Collection<Map<String, ?>>` (I would be pretty surprised if anyone had a non-String key, but it also touches every provider soooooo yeah)